srcpkgs/sway/template

```diff
@@ -1,6 +1,7 @@
 # Template file for 'sway'
 pkgname=sway
-version=1.7
+_rc="1"
+version=1.8
```

Comment:
according to the [changelog](https://github.com/swaywm/sway/issues/7019#issue-1244738993):

> On Linux, CAP_SYS_NICE can be set to improve scheduling

Is this something we want or do we want to leave this to the user?

Reply:
I think something we want, kwin has it: https://github.com/void-linux/void-packages/blob/ded277952d79804a1a4ed473e5848f65fe121bda/srcpkgs/kwin/INSTALL

Reply:
added

Reply:
also add libcap-progs to $depends

Reply:
done